src/api/controllers/getAuthorsController.js

```diff
@@ -0,0 +1,46 @@
+import models from '../models';
+import status from '../../helpers/constants/status.codes';
+import generatePaginationDetails from '../../helpers/generate.pagination.details';
+
+const { Article } = models;
+const { User } = models;
+/**
+ *
+ *
+ * @export
+ * @class getAuthors
+ */
+export default class getAuthors {
```

Comment:
@Ramadhan0 the class name should be a noun e.g `AuthorsController` and the method name should be plural like `getAuthors`(because you're fetching a list of authors). Follow the convention your team-mates have been using.

Reply:
@Ramadhan0 I can't see that you have implemented this feedback